app/src/main/java/io/homeassistant/companion/android/widgets/button/ButtonWidget.kt

```diff
@@ -228,6 +233,10 @@ class ButtonWidget : AppWidgetProvider() {
             // Render the icon into the Button's ImageView
             setImageViewBitmap(R.id.widgetImageButton, icon.toBitmap(width, height))
 
+            widget?.iconColor?.let {
+                setCustomColorToIcon(it, this)
```

Comment:
This will overwrite the color set to the icon in line 204/206, which is fine if you have set a custom color but when using the transparent theme it might create a mismatch (black icon on a colored circle with white text, while the option says it'll color text + icon, and it just looks odd).
![Transparent widget: blue circle with black icon, and white text below](https://github.com/user-attachments/assets/aa2ee548-0c89-4fa6-98fa-dbd15e3021df)